pkg/payment/refund/request.go

```diff
@@ -0,0 +1,6 @@
+package refund
+
+// Request represents a request for creating a refund.
+type Request struct {
+	Amount float64 `json:"amount"`
```

Comment:
```suggestion
	Amount float64 `json:"amount, omitempty"`
```
Imagino que o campo do request deve ser populado